po/es.po

```diff
@@ -201,7 +201,8 @@ msgstr ""
 msgid ""
 "Save the \"Filter By…\" settings by enabling the \"Save Filter\" checkbox."
 msgstr ""
-"Guarda las opciones «Filtrar por...» habilitando la casilla «Guardar Filtro»."
+"Guarda los ajustes de \"Filtrar Por...\" activando la casilla de \"Guardar "
```

Comment:
@cyphra you don't like the comillas latinas? IMHO they look nicer than dumb ASCII quotation. If you do not know their keycodes, the weblate interface has them over the edit field.
However you decide, you should be consistent in your writing.